src/controllers/mainController.ts

```diff
@@ -288,35 +289,40 @@ export default class MainController implements vscode.Disposable {
         });
     }
 
+    /**
+     * Helper function to toggle SQLCMD mode
+     */
+    private async toggleSqlCmdMode(isSqlCmd: boolean): Promise<boolean> {
+        return this._outputContentProvider.toggleSqlCmd(this._vscodeWrapper.activeTextEditorUri).then(async () => {
+            await this._connectionMgr.onChooseLanguageFlavor(true, !isSqlCmd);
+            return Promise.resolve(true);
+        });
+    }
+
+
     /**
      * Handles the command to enable SQLCMD mode
      */
     private async onToggleSqlCmd(): Promise<boolean> {
         let isSqlCmd: boolean;
```

Comment:
The only place this is called doesn't even use the return value so I don't really see the point in having it